apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/components/ApplicationInformation/ApplicationInformation.tsx

```diff
@@ -113,7 +125,7 @@ interface ApplicationInformationProps {
   applicationQuery: FragmentType<
     typeof ApplicationInformation_PoolCandidateFragment
   >;
-  snapshot: User; // recreated from Json
+  snapshot: User & { bilingualEvaluation?: BilingualEvaluation }; // recreated from Json
```

Comment:
Is this because of snapshot versioning?  If so, I think we need to approach this in a more extensible way.  Maybe multiple copies of the component with different props for different versions?  Since the snapshot isn't actually coming from the API directly is it every worth doing fragments?  Or just using them as types?

Reply:
Huh, I think we may need to dig deeper into the snapshot display again :cry: 

This is very similar to the other comment about the casting. The problem here is that the component that has the fragment is identical to the one we want to display here visually (unless of course you have an out of date snapshot).

So the fragments are very helpful when we can use them (showing the current profile) but get trickier when we cant (within the application).

Maybe we just need two versions, a set of components that render snapshot data with all the special handling for versions and legacy data and another version that is leaner and just hanldes the current user data model :thinking: